packages/design-system/src/components/N8nMenuItem/MenuItem.vue

```diff
@@ -30,6 +30,7 @@
 					[$style.disableActiveStyle]: !isItemActive(child),
 					[$style.active]: isItemActive(child),
 				}"
+				:data-test-id="`menuitem-${child.id}`"
```

Comment:
I'm not a huge fan of adding unique ids to the test-id identifier. While it's easier to target an element this way, it prevents us from getting all `menuitem` if we'd need to count them or iterate over them in any other way. To get the individual menu item in cypress could be done like `cy.getByTestId('menuitem-credentials').contains($id).parents('[data-test-id="workflow-card"]')`

Reply:
but the id here is `credentials` so the test id is `menuitem-credentials`